### PR TITLE
fix(activity-feed-zoom): add a min-height prop to the items container

### PR DESCRIPTION
**Pull request by dsporysz-box**
The activity feed with comments list doesn't scale with zoom correctly. At 400% zoom, the top header and the comment form cover the whole screen making the comment history not visible and it's not possible to scroll it into a view.

This PR adds a min-height property to the comments container so it does not get squished to zero height. The min-height is the same as the comment form.

Before:
zoom400before
After:
zoom400after

### CODE REVIEW

**Review by bkepka-box — DISMISSED** [review]

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 1]
```diff
@@ -49,6 +49,7 @@
         flex: 1 1 auto;
         flex-direction: column;
         width: 100%; // Explicit width required since flex-direction is column
+        min-height: 93px;
```
Comment (loonskai) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:52]: May be a good candidate for a CSS variable

**Review by loonskai — DISMISSED** [review]

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss (hunk 1 above)
Comment (dsporysz-box) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss]: Thanks, I added a var

**Review by loonskai — APPROVED** [review]

**Review by bkepka-box — APPROVED** [review]

**Review by tjuanitas — APPROVED** [review]

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 2]
```diff
@@ -49,6 +49,7 @@
         flex: 1 1 auto;
         flex-direction: column;
         width: 100%; // Explicit width required since flex-direction is column
+        min-height: $sidebarContentMinHeight;
```
Comment (tjuanitas) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:52]: I didn't notice this before but I see now there's two vertical scroll bars that would appear at the same time

multiple scroll bars in the same axis can have some unintuitive behavior to users, is that okay? is that an accessibility concern?

**Review by tjuanitas — COMMENTED** [review]
sorry I had a thought come to mind after I approved

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss (hunk 2 above)
Comment (dsporysz-box) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:52]: @tjuanitas I've reached out to the Product Team and this solution has been accepted. Two scroll bars appear only at higher zoom and this solution lets us avoid impacting the broader user experience. Great attention to the details! Thanks!

Comment (tjuanitas) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:52]: great, sounds good to me!

**Pull request merged**